src/avro-derive/Cargo.toml

```diff
@@ -5,12 +5,10 @@ authors = ["Brennan Vincent <brennan@umanwizard.com>"]
 license = "Apache-2.0"
 edition = "2018"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
did you do this or does cargo-maniformat strip comments?

Reply:
oh nvm I see a comment is preserved below.

Reply:
It's... complicated: https://github.com/benesch/cargo-manifmt#limitations

In this case though getting rid of this comment seems correct. No point in having this clutter around.

Reply:
Yeah I agree with deleting that one! Seems fine for something you're going to run occasionally. Thanks!